docs/reference/modules/transport.asciidoc

```diff
@@ -88,19 +88,22 @@ example above:
    determines the time in seconds that a connection must be idle before
    starting to send TCP keepalive probes.
    Only applicable on Linux and Mac, and requires JDK 11 or newer.
-   Defaults to -1, which does not set this option at the socket level, but
-   uses default system configuration instead.
+   Defaults to 60 on applicable configurations, and -1 otherwise, which does
+   not set this option at the socket level, but uses the default system
+   configuration instead.
 * `tcp.keep_interval`: Configures the `TCP_KEEPINTVL` option for this socket,
    which determines the time in seconds between sending TCP keepalive probes.
    Only applicable on Linux and Mac, and requires JDK 11 or newer.
-   Defaults to -1, which does not set this option at the socket level, but
-   uses default system configuration instead.
+   Defaults to 10 on applicable configurations, and -1 otherwise, which does
```

Comment:
I would have chosen `60` for the default here I think. The purpose of these keepalives is more to tell any intermediate devices that the connection remains alive rather than to test the health of the connection ourselves, and I don't think that needs a packet every 10s. IMO #59222 is a better way to pick up on failed connections much more quickly in cases where it matters.

That said, anything less than the ridiculous default of 7200 is a major improvement anyway.